add_to_app/prebuilt_module/flutter_module/README.md

```diff
@@ -0,0 +1,14 @@
+# flutter_module
+
+An example Flutter module used in the Flutter add-to-app samples. For more
```

Comment:
For this readme and the other ones from existing projects that were just copied, should they just now point to the parent directory instead of the root?

Reply:
done